static/js/biomedical/disease/disease_chart.test.tsx

```diff
@@ -175,3 +176,26 @@ test("getCompoundDiseaseContraindication", () => {
     }
   }
 });
+
+test("getDiseaseCommonName", () => {
+  const cases: {
+    data: GraphNodes;
+    wantArray: string;
```

Comment:
nit: `wantName` instead of `wantArray` since it's not expecting an array?

Reply:
RIght! Changed to `wantName`